superset/views/core.py

```diff
@@ -629,16 +630,19 @@ class DashboardAddView(DashboardModelView):  # noqa
 appbuilder.add_view_no_menu(DashboardAddView)
 
 
+@talisman(force_https=False)
```

Comment:
Assuming these are needed so that some internal health check passes with TLS being handled by a LB or reverse proxy... 👍